CI, PHP CS fixer fails because of new PHP version 8.1

### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add screenshots

This issue has been discussed on slack. I just put it here for information.

Since about 6 hours ago, PHP CS fixer fails because of new PHP version 8.1. See  e.g. https://github.com/PrestaShop/PrestaShop/runs/4324452528?check_suite_focus=true.

Error:
```
Run ./vendor/bin/php-cs-fixer fix --dry-run --diff
PHP needs to be a minimum version of PHP 5.6.0 and maximum version of PHP 8.0.*.
Current PHP version: 8.1.0.
To ignore this requirement please set `PHP_CS_FIXER_IGNORE_ENV`.
If you use PHP version higher than supported, you may experience code modified in a wrong way.
Please report such cases at https://github.com/FriendsOfPHP/PHP-CS-Fixer .
Error: Process completed with exit code 1.
```

### Expected behavior

PR should validate if they are valid.

### Steps to reproduce

Submit a PR

### PrestaShop version(s) where the bug happened

Prestashop/Prestashop

### PHP version(s) where the bug happened

_No response_

### If your bug is related to a module, specify its name and its version

_No response_

--- a/.github/workflows/php.yml
+++ b/.github/workflows/php.yml
@@ -13,6 +13,7 @@ jobs:
       - name: Setup PHP
         uses: shivammathur/setup-php@v2
         with:
+          php-version: 8.0
           extensions: mbstring, intl, gd, xml, dom, json, fileinfo, curl, zip, iconv
       - uses: actions/checkout@v2
 

